test/outdated/outdated_test.dart

```diff
@@ -450,7 +450,9 @@ Future<void> main() async {
 
     await variations('handles_sdk_dependencies', environment: {
       'FLUTTER_ROOT': d.path('flutter-root'),
-      '_PUB_TEST_SDK_VERSION': '2.13.0'
+      '_PUB_TEST_SDK_VERSION': '2.13.0',
+      // To test that the reproduction command is reflected correctly.
+      'PUB_ENVIRONMENT': 'flutter_cli',
```

Comment:
technically it's probably something like;
```suggestion
      'PUB_ENVIRONMENT': 'flutter_cli:get',
```